Tests/KituraSampleRouterTests/KituraTest.swift

```diff
@@ -41,18 +41,22 @@ extension KituraTest {
 
     func performServerTest(asyncTasks: @escaping (XCTestExpectation) -> Void...) {
         let router = RouterCreator.create()
-        Kitura.addHTTPServer(onPort: 8090, with: router)
-        Kitura.start()
-        sleep(1)
```

Comment:
As of Kitura-1.1.1, all we should need to do is remove this `sleep(1)` call since we are not dependent on the lifecycle started hook for this. When Kitura.start() returns, the socket is already open to accept connections for the tests that follow.
As this is meant as a guide to help people starting out with Kitura, it maybe better to use the started/stopped callbacks for tests that actually test the lifecycle stage hooks.